satpy/etc/readers/viirs_l2.yaml

```diff
@@ -0,0 +1,126 @@
+reader:
+  name: viirs_l2
+  short_name: VIIRS L2
+  long_name: SNPP VIIRS Level 2 data in netCDF4 format
+  description: Generic NASA VIIRS L2 Reader
+  status: Alpha
+  supports_fsspec: false
+  reader: !!python/name:satpy.readers.yaml_reader.GEOFlippableFileYAMLReader
+  sensors: [viirs]
+  default_datasets:
+
+file_types:
+  cldprop_l2_viirs:
+    file_reader: !!python/name:satpy.readers.viirs_l2.VIIRSL2FileHandler
+    file_patterns:
+      - 'CLDPROP_L2_VIIRS_{spacecraft_name:s}.A{start_time:%Y%j.%H%M}.{collection:03d}.{production_time:%Y%j%H%M%S}.nc'
+  cldmsk_l2_viirs:
+    file_reader: !!python/name:satpy.readers.viirs_l2.VIIRSL2FileHandler
+    file_patterns:
+      - 'CLDMSK_L2_VIIRS_{spacecraft_name:s}.A{start_time:%Y%j.%H%M}.{collection:03d}.{production_time:%Y%j%H%M%S}.nc'
+  aerdb_l2_viirs:
+    file_reader: !!python/name:satpy.readers.viirs_l2.VIIRSL2FileHandler
+    file_patterns:
+      - 'AERDB_L2_VIIRS_{spacecraft_name:s}.A{start_time:%Y%j.%H%M}.{collection:03d}.{production_time:%Y%j%H%M%S}.nc'
+  aerdb_l2_viirs_nrt:
+    file_reader: !!python/name:satpy.readers.viirs_l2.VIIRSL2FileHandler
+    file_patterns:
+      - 'AERDB_L2_VIIRS_{spacecraft_name:s}.A{start_time:%Y%j.%H%M}.{collection:03d}.nrt.nc'
```

Comment:
Unless you expect people to provide the AERDB production/archive files and the NRT files at the same time, these file types could be combined by moving the NRT file pattern to the `file_patterns` of the non-NRT file type.